src/class-parsely.php

```diff
@@ -793,6 +796,29 @@ public function display_admin_warning() {
 		<?php
 	}
 
+	/**
+	 * Display a dismissible admin warning if the current WordPress or PHP versions are below the required minimum for the 3.0 release of wp-parsely
+	 * We should get rid of this warning when we release 3.0
+	 *
+	 * @category Function
+	 * @package Parsely
+	 */
+	public function display_admin_upgrade_warning() {
+		global $wp_version;
+		if ( version_compare( phpversion(), '7.1', '>=' ) && version_compare( $wp_version, '5.0', '>=' ) ) {
+			return;
+		}
+
+		$message = sprintf(
+		/* translators: %s: Plugin settings page URL */
+			__( '<strong>The next version of the Parse.ly plugin will not work with the current setup.</strong> WordPress 5.0 and PHP 7.1 will be the new required minimum versions. You can read more <a href="%s">here</a>.', 'wp-parsely' ),
```

Comment:
```suggestion
			__( '<strong>The next version of the Parse.ly plugin will not work with the current setup.</strong> WordPress 5.0 and PHP 7.1 will be the <a href="%s">new required minimum versions</a>.', 'wp-parsely' ),
```

We don't need a whole sentence just to add a "click here" (bad for accessibility).